src/utils/get-port.ts

```diff
@@ -5,6 +5,7 @@ interface IPorts {
 export function getPort(platform: string): number {
   const ports: IPorts = {
     static: 80,
+    python: 80,
```

Comment:
Should we define a default port? Shouldn't we ask for a port each time?

Reply:
I assumed most python users just want to deploy a script that runs without needing an HTTP port, so I set the default port. However, if you think it's better to ask the port each time and not to set a default port, I will change it.
Can't users change the port using the `liara.json` file?

Reply:
As I’ve observed, most Python users simply deploy scripts using `while true` loops. Setting a default port simplifies deployment for them.